samples/step-by-step/sample.md

```diff
@@ -7,100 +7,191 @@ redirects:
 - nservicebus/nservicebus-step-by-step-guide
 ---
 
-In this sample shows a very simple ordering system that
+This sample illustrates basic NServiceBus concepts in a step-by-step fashion. Alternatively, the completed sample code can be downloaded above.
 
- * sends a command from a client to a server
- * that server handles the command and publishes a new event about the success
- * a subscriber listens to, and handles, the event
 
+## Concepts
 
-## The Shared Project
+The goal of this step-by-step sample is to show how to build a simple system that uses messaging to:
 
-The `Shared` project is the container for shared classes including message definitions. This project will be shared between the client and server so both sides agree on the typed message descriptions. It is referenced by all projects in the solution.
+ * Send a command from a client to a server
+ * Receive a message from a client on the server that publishes a new event
+ * Add a new subscriber that can receive published events
 
+Completing these steps will serve as an introduction to many important NServiceBus concepts:
 
-### PlaceOrder Command
+* Basic project setup
+* Self-hosting NServiceBus within a console application
+* Defining and sending commands
+* Handling messages
+* Defining and publishing events
+* Logging
 
-Used to initiate an order
 
-snippet:PlaceOrder
+## Project structure
 
+The finished solution will contain four projects. Create a new Visual Studio solution and create the following:
 
-### OrderPlaced Event
+* A Console Application named `Client`: The client's job will be to send `PlaceOrder` commands to the Server.
+* A Console Application named `Server`: The server's job will be to handle the `PlaceOrder` command, and then publish an `OrderPlaced` event to any interested subscribers.
+* A Console Application named `Subscriber`: The subscriber will subscribe to 
+* A Class Library named `Shared`.
 
-Used to indicate that an order has been received and processed.
+The client's job will be to send `PlaceOrder` commands to the server. The server will process the `PlaceOrder` commands, and then publish an `OrderPlaced` event to any interested subscribers. The subscriber will subscribe to `OrderPlaced` events published from the server, and process those events when they arrive.
 
-snippet:OrderPlaced
+The shared project will be referenced by all of the applications and serve as a central location to store message definition classes and common pieces of configuration. This is not a best practice to combine all these things in one place, but serves to illustrate how things work for this simple example.
+
+
+### Setting dependencies
+
+Before beginning, set up the necessary dependencies.
+
+* Add a reference to the `Shared` project to the `Client`, `Server`, and `Subscriber` projects.
+* In each project, add a reference to the `NServiceBus` NuGet package, either using the Package Manager Console or the NuGet Package Manager visual tool.
```

Comment:
", either using the Package Manager Console or the NuGet Package Manager visual tool." i think this is redundant give our target audience